server/src/game/GameData.ts

```
@@ -214,13 +214,15 @@ class CustomSingleplayerGameData extends GameData {
     }
     super(owner, gameMode);
     // This assumes that data has already been validated.
-    this.baseHealth = settings.baseHealth;
-    this.maximumBaseHealth = settings.baseHealth;
-    this.clocks.comboReset.actionTime = settings.comboTime;
-    this.enemySpeedCoefficient = settings.enemySpeedCoefficient;
-    this.clocks.enemySpawn.actionTime = settings.enemySpawnTime;
-    this.enemySpawnThreshold = settings.enemySpawnThreshold;
-    this.clocks.forcedEnemySpawn.actionTime = settings.forcedEnemySpawnTime;
+    this.baseHealth = parseFloat(settings.baseHealth);
```

Biome

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)